README.md

```diff
@@ -1,4 +1,108 @@
 # OpenFeature project governance
 
-> :warning: Details are coming soon!
+[![Roadmap](https://img.shields.io/static/v1?label=Roadmap&message=public&color=green)](https://github.com/orgs/openfeatureflags/projects/1)
+[![Contributing](https://img.shields.io/static/v1?label=Contributing&message=guide&color=blue)](https://github.com/openfeatureflags/.github/blob/main/CONTRIBUTING.md)
+
+This document describes the bootstrap governance process under which the project will operate
+until the final governance process is identified.
+
+> :warning: This is a temporary (aka bootstrap) governance document that
+> is effective until the project is fully established.
+> See [this issue](https://github.com/openfeatureflags/governance/issues/11) for the scope of the full governance document.
+
+## Project manifesto
+
+### Goal
+
+Our goal is to create an open standard for feature flag management to support a robust feature flag ecosystem using cloud native technologies.
+
+### Scope
+
+The scope includes but not limited to:
+
+- An **open standard** that allows adopters and vendors to easily integrate with a spec-compliant feature flag solutions
+- A **vendor-agnostic ecosystem** for users, application developers, SREs, and vendors
+- Unified API and SDK
+- Developer-first, cloud-native reference implementation
+- Extensibility for open source and commercial offerings
+- Adoption by the open source and proprietary projects listed on the [Cloud Native Landscape](https://landscape.cncf.io/)
+
+### Why OpenFeature?
+
+There are many existing solutions, open source or not.
+Unfortunately, they lack key features needed for wide adoption
+by companies and in the open source ecosystem:
+
+- **Many SDKs** doing the same thing, but slightly differently
+- Many companies **"rebuilding the wheel"**
+- SDKs are generally open source but have no ecosystem
+- **Vendor lock-in** at code-level
+- **No collaboration** across vendors 
+- **Missing integration** with observability tools (e.g. OpenTelemetry)
+
+### Project Differentiators
+
+This project is more than just copy and paste of existing solutions
+
+* Current solutions often have **eval logic in SDKs**.
+  This may require **updating all your apps** for a new rule type.
+* Config in Feature Flagging tools that are **separate from the rest of the app** make it harder to use approaches like GitOps. 
+* Config and rule changes are **hard to integrate into observability** tools 
+* Information to decide on features already available in observability tools is **not leveraged and collected twice**
+* No **agreed on grammar/structure** for rules which could be automatically validated (e.g. with CUE)
```

Comment:
```suggestion
* Current solutions often have **eval logic in SDKs**.
  This may require **updating all your apps** for a new rule type.
* Config in Feature Flagging tools that are **separate from the rest of the app** make it harder to use approaches like GitOps. 
* Config and rule changes are **hard to integrate into observability** tools 
* Information to decide on features already available in observability tools is **not leveraged and collected twice**
* No standardized format for defining feature flagging rules
```